spec/openai/assistant_vcr_spec.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+# rubocop:disable Style/GlobalVars
+
+RSpec.describe Openai::Assistant do
+  $assistant = nil # keep the response after create for another method like retrieve, delete,...
```

Comment:
Please take a look on `let`, `let!`, `let_it_be`

Reply:
thanks anh, I was updated to `let` to hold the response